app/channels/notifications_channel.rb

```diff
@@ -0,0 +1,9 @@
+class NotificationsChannel < ApplicationCable::Channel
+  def subscribed
+    stream_for current_user
+  end
+
+  def unsubscribed
+    stop_all_streams
```

Comment:
why it's needed?

Reply:
@igorkasyanchuk  Unsubscribes all streams associated with this channel from the pubsub queue.